### DISCUSSION TITLE
[Feature] Set default user settings as admin

### DISCUSSION THREAD

**Discussion by bo0tzz** [immich-app/immich#13651] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

As an admin, it would be nice to be able to decide on the default user settings for things like feature flags, locale, or notifications.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by JonahKr** [comment]
This is especially useful when supporting Mail Notifications.
While collecting images from multiple people in a shared album, by default everyone will get spammed with Mails.

**Comment by Liziloo** [comment]
In the name of Popeye the Sailor, please change this. I didn't realize when I started a shared album that it would spam collaborators every time I added photos. I'm archiving family photos going back hundreds of years. They're getting 60 emails/day.

**Comment by jdaleo23** [comment]
Has there been any progress on this? I just started a server and installed the app on my family's phones, and instead of having to go to each of their phones individually to set up every detail, it would be nice to be able to do it as the admin and push it down as the default options.

**Comment by jenda69** [comment]
I just moved domains and it is quite annoying that half of my settings are now wrong because defaults are not what I'd like. And they are saved in cookies/localstorage so I have to redo them on all my devices and then go through wife's devices too...

**Comment by leckye** [comment]
Anyone knows a way to overrides default value defined in web/src/lib/stores/preferences.store.ts?

is modifed user settings stored in table user_metadata?

**Comment by MrPistachios** [comment]
Now that dark mode is hidden in settings, it would be nice to have a server wide option to have dark mode be the default for the web browser, or a way to permanently set the toggle back to where it was so we don't have to change it each time we open the browser if we have browser fingerprinting disabled 

<img width="902" height="409" alt="Screenshot_20260708_105836" src="https://github.com/user-attachments/assets/ce96ffe7-0439-4551-bcf0-c5c775b0f6f4" />
